Resources/Catalog/LcgFileCatalogClient.py

```diff
@@ -2,13 +2,13 @@
 
 """
 import DIRAC
-from DIRAC                                                    import S_OK, S_ERROR, gLogger, gConfig
-from DIRAC.Resources.Utilities                                import checkArgumentFormat
-from DIRAC.Resources.Catalog.FileCatalogueBase                import FileCatalogueBase
-from DIRAC.Core.Utilities.Time                                import fromEpoch
-from DIRAC.Core.Utilities.List                                import breakListIntoChunks
-from DIRAC.Core.Security.ProxyInfo                            import getProxyInfo, formatProxyInfoAsString
-from DIRAC.ConfigurationSystem.Client.Helpers.Registry        import getDNForUsername, getVOMSAttributeForGroup, getVOForGroup, getVOOption
+from DIRAC                                                 import S_OK, S_ERROR, gLogger, gConfig
+from DIRAC.DataManagementSystem.Utilities.CatalogUtilities import checkCatalogArguments
+from DIRAC.Core.Utilities.Time                             import fromEpoch
```

Comment:
It was there already anyway... We should rediscuss the need for these utilities and keep only added value to existing python modules... fromEpoch may be one of them...
This being said the name is not good and we should avoid file names that clash with python modules, as it is a pain on file systems that are not case sensitive :(